src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -1100,12 +1118,15 @@ HRESULT AtlasEngine::_createTextFormat(const wchar_t* fontFamilyName, DWRITE_FON
     // at a base scale of 72 DPI, whereas DirectWrite uses 96 DPI.
     // --> Multiply by 1.333... to convert from 72 DPI to 96 DPI scale.
     fontSize *= 4.0f / 3.0f;
+    fontSize = 21.8259335;
 
     const auto hr = _sr.dwriteFactory->CreateTextFormat(fontFamilyName, nullptr, fontWeight, fontStyle, DWRITE_FONT_STRETCH_NORMAL, fontSize, L"", textFormat);
     if (SUCCEEDED(hr))
     {
         const auto tf = *textFormat;
-        tf->SetTextAlignment(DWRITE_TEXT_ALIGNMENT_CENTER);
+        // We can't set DWRITE_TEXT_ALIGNMENT_CENTER

```

Comment:
comment nit: // explain why

Reply:
Removed because it didn't pan out anyways.